packages/core/src/components/dropdown-menu/dropdown-menu.stories.ts

```diff
@@ -0,0 +1,16 @@
+import { storiesOfDropdownMenu } from "@dso-toolkit/sources";
+import { storiesOf } from "@storybook/web-components";
+
+import { dropdownMenuTemplate } from "./dropdown-menu.template";
+import readme from "./readme.md";
+
+storiesOfDropdownMenu(
+  {
+    module,
+    storiesOf,
+    readme,
+  },
+  {
+    dropdownMenuTemplate: dropdownMenuTemplate,
```

Comment:
Hier kunnen we de korte notatie gebruiken

Reply:
Aangepast